### ISSUE TITLE
Fails to init if NODE_PATH is set and has more than one entry

### ISSUE THREAD

**Issue by crccheck** [dylang/npm-check#263]
Today I started running into:
```
$ npm-check -sug
process.env.NODE_PATH does not exist. Please check the environment variables.
For more detail, add `--debug` to the command
```

But I do have `NODE_PATH` set, what's going on?
```
$ env | grep NODE_PATH
NODE_PATH=/usr/local/lib/node_modules:/Users/crccheck/local/node
```

It turns out it also checks to make sure `NODE_PATH` exists on the filesystem

https://github.com/dylang/npm-check/blob/1ac1eb3455eb02d2a4265d640a7842ead207f9d4/lib/state/init.js#L17

But it according to the [Node docs](https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders), `NODE_PATH` can be "a colon-delimited list of absolute paths". So the current logic should split all the paths and check them, maybe with a `.reduce`.

----

I don't know why this just started today. I've had that in my environment for years. The only change I did was to upgrade to Node 8 LTS from Node 8.8. I fixed the problem by removing `NODE_PATH` from my dotfiles because I wasn't even using it but others may run into this.

**Comment by crccheck** [comment]
ohh, I guess I upgraded npm-check and got #247 
The git blame for that line was three months ago so I thought it was in npm-check since then instead of just yesterday.

**Comment by LinusU** [comment]
Sorry about this, working on fix now...

**Comment by LinusU** [comment]
Fix is up at #264, will merge as soon as tests pass, if someone has time to try it out in the meantime, it would be highly appreciated

It should be installable by running `npm i -g dylang/npm-check#0c674fb`

**Comment by LinusU** [comment]
Fixed by #264 in 5.5.1
